### PR TITLE
V2 Workflows: Intital set of docs

### PR DESCRIPTION
**Pull request by rohit-nayak-ps**
Signed-off-by: Rohit Nayak <rohit@planetscale.com>

### CODE REVIEW

File: content/en/docs/reference/vreplication/v2/cancel.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Cancel
+description: Tear down a workflow where no traffic has been switched
+weight: 90
+---
+##### _Experimental_
+This documentation is for a new (v2) set of vtctld commands. See [RFC](https://github.com/vitessio/vitess/issues/7225) for more details.
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/cancel.md:7]: vtctl or vtctld? I think we just need to be consistent throughout.

File: content/en/docs/reference/vreplication/v2/complete.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+</div>
+
+#### -rename_tables
+**optional**\
+**default** false
+
+<div class="cmd">
+The rename_tables flag is applicable only for MoveTables. Tables are renamed instead of being deleted. Currently the new name is _&lt;table_name&gt;.
+<br/><br/>
+(We plan to change this to use the logic used by `ptosc` using the template _&lt;table_name&gt;_old. The new naming convention has the additional benefit that vreplication will automatically ignore such tables from getting resharded or streamed by the VStream API)
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/complete.md]: The official name for `ptosc` is `pt-online-schema-change`, though I do see people referring to it as `pt-osc` as well.

File: content/en/docs/reference/vreplication/v2/movetables.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+### Common use cases for MoveTables
+
+#### Adopting Vitess
+
+For those wanting to try out Vitess for the first time MoveTables provides an easy way to route part of their workload
+to Vitess with the ability of migrating back at any time without any risk. You point a vttablet to your existing MySQL installation, spin up a unsharded Vitess cluster and use a MoveTables workflow to start serving some tables from Vitess. You can also go further and use a Reshard workflow to experiment with a sharded version of a part of your database.
+
+See [user guide](../../../../../docs/user-guides/configuration-advanced/unmanaged-tablet/#move-legacytable-to-the-commerce-keyspace) for detailed steps
+
+#### Horizontal Sharding
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/movetables.md]: Vertical

File: content/en/docs/reference/vreplication/v2/reversetraffic.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Command
+
+```
+MoveTables/Reshard -v2 [-tablet_types <tablet_types_csv>] [-cells <cells>] ReverseTraffic <targetKs.workflow>
+```
+
+### Description
+
+`ReverseTraffic` switches traffic in the reverse direction for the tablet_types specified. The traffic should have been previous switched forward using SwitchTraffic for the cells/tablet_types specified.
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/reversetraffic.md]:
```suggestion
`ReverseTraffic` switches traffic in the reverse direction for the tablet_types specified. The traffic should have been previously switched forward using SwitchTraffic for the cells/tablet_types specified.
```

File: content/en/docs/reference/vreplication/v2/start.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Reshard -v2 [-source_shards=<source_shards>] [-target_shards=<target_shards>]
+  [-cells=<cells>] [-tablet_types=<source_tablet_types>]  [-skip_schema_copy]
+  Start <keyspace.workflow>
+
+```
+
+### Description
+
+`MoveTables/Reshard Start` initiates a new workflow. The workflow name should not clash with that of an existing workflow.
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/start.md]:
```suggestion
`MoveTables/Reshard Start` initiates a new workflow. The workflow name should not conflict with that of an existing workflow.
```

File: content/en/docs/reference/vreplication/v2/start.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+**optional**\
+**default** local cell
+
+<div class="cmd">
+A comma separated list of cell names or cell aliases. This list is used by VReplication to determine which
+cells should be used to pick a tablet for selecting data from the source keyspace.<br><br>
+
+###### Uses
+
+* Improve performance by using picking a tablet in cells in network proximity with the target
```
Comment (deepthi) [inline content/en/docs/reference/vreplication/v2/start.md]:
```suggestion
* Improve performance by picking a tablet in cells in network proximity with the target
```

File: content/en/docs/reference/vreplication/v2/cancel.md (hunk 1 above)
Comment (rohit-nayak-ps) [reply content/en/docs/reference/vreplication/v2/cancel.md:7]: Used vtctld

File: content/en/docs/reference/vreplication/v2/complete.md (hunk 2 above)
Comment (rohit-nayak-ps) [reply content/en/docs/reference/vreplication/v2/complete.md]: Fixed

File: content/en/docs/reference/vreplication/v2/reversetraffic.md (hunk 4 above)
Comment (rohit-nayak-ps) [reply content/en/docs/reference/vreplication/v2/reversetraffic.md]: fixed

File: content/en/docs/reference/vreplication/v2/start.md (hunk 5 above)
Comment (rohit-nayak-ps) [reply content/en/docs/reference/vreplication/v2/start.md]: fixed

File: content/en/docs/reference/vreplication/v2/start.md (hunk 6 above)
Comment (rohit-nayak-ps) [reply content/en/docs/reference/vreplication/v2/start.md]: fixed

**Review by sougou — APPROVED** [review]
Not merging this, in case you decide to piggy-back the `Create` rename. Feel free to merge when you see fit.

**Comment by sougou** [comment]
Doh. I see that you've already renamed.

**Pull request merged**